tests/Rules/Validation/ScopeIlluminateRequestValidateMethodTest.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+namespace Rules\Validation;
+
+use Hihaho\PhpstanRules\Rules\Validation\ScopeIlluminateRequestValidateMethods;
+use PHPStan\Analyser\Error;
+use PHPStan\Rules\Rule;
+use PHPStan\Testing\RuleTestCase;
+use PHPUnit\Framework\Attributes\Test;
+
+/**
+ * @extends RuleTestCase<ScopeIlluminateRequestValidateMethods>
+ */
+final class ScopeIlluminateRequestValidateMethodTest extends RuleTestCase
+{
+    protected function getRule(): Rule
+    {
+        return new ScopeIlluminateRequestValidateMethods();
+    }
+
+    #[Test]
+    public function illuminate_http_request_does_not_use_unvalidated_methods_outside_app_http_requests_namespace(): void
+    {
+        /** @var Error[] $errors */
+        $errors = $this->gatherAnalyserErrors([__DIR__ . '/../../stubs/App/Http/Controllers/PeopleControllerStub.php']);
+        $validErrors = array_filter(
+            $errors,
+            static fn (Error $error): bool => $error->getIdentifier() === 'hihaho.request.unsafeRequestData'
+        );
+        self::assertCount(6, $validErrors);
+
+        $unsafeRequestDataError = static fn (int $line) => [
```

Comment:
because of the amount of the repeated error messages, I've extracted it to a simple closure